src/main/lrsql/util/oidc.clj

```diff
@@ -1,19 +1,187 @@
 (ns lrsql.util.oidc
   "OpenID Connect Utilities"
```

Comment:
Going off of the above and looking more closely at the functions in this namespace, I think `get-configuration`, `resource-interceptors`, `resolve-authority-claims`, and `make-authority-fn` could be moved to a `lrsql.init.oidc` namespace, leaving `parse-scope-claim` and `token-auth-identity` in this namespace.

Reply:
Sure, that sounds fine

Reply:
see https://github.com/yetanalytics/lrsql/pull/205/commits/50247e22222e58a8d11cbb94e804a599dbc17a0a